fpmsyncd/routesync.cpp

```diff
@@ -920,3 +920,31 @@ string RouteSync::getNextHopIf(struct rtnl_route *route_obj)
 
     return result;
 }
+
+// Check if ependent entries are re-conciled 
+bool RouteSync::isReadyToReconcile()
+{
+    vector<string> required_modules = {
+            "orchagent",
+        };
+
+    for(string& module : required_modules)
+    {
+        WarmStart::WarmStartState state;
+        
+        WarmStart::getWarmStartState(module, state);
+        if(state == WarmStart::RECONCILED || state == WarmStart::WSDISABLED)
+        {
+            SWSS_LOG_INFO("Module %s Reconciled %d",module.c_str(), (int) state);            
+        }
+        else
+        {
+            SWSS_LOG_INFO("Module %s NOT Reconciled %d",module.c_str(), (int) state);            
+            //return false;
+            //Return true untill dependent module code is commited
```

Comment:
what is the dependent module code? Are there any further changes expected?

Reply:
Dependent code mease all the code PRs which add this dependency check. Since these are committed as different PRs, I have stubbed the actual check for dependency. This will ensure that if one of the PR is not present, the dependency check will not fail. Once all the PRs related to this dependence check are merged, these stubbs will be deleted and actual check will be enabled.

Reply:
same applies to the above comment regarding orchagent code too. The orchagent code is already present. Once all the PRs ( now only 1556 is remaining) are merged, the actual code check will be activated

Reply:
will commit the actual code and remove the stubbed code as other PRs are already merged and this is the last PR